### DISCUSSION TITLE
What am I doing wrong with AutoRespond

### DISCUSSION THREAD

**Discussion by yarosh69** [ORelio/Minecraft-Console-Client#2372] [Q&A] [answered]
![ScreenShot_20221210161010](https://user-images.githubusercontent.com/120141684/206856914-b669f4b7-68af-439c-b73e-928e1ed346ca.png)
![ScreenShot_20221210160749](https://user-images.githubusercontent.com/120141684/206856915-0b6a838e-04df-4b37-8c38-8f89f7085b9c.png)
![ScreenShot_20221210160813](https://user-images.githubusercontent.com/120141684/206856916-0afded0f-d75c-4e50-a326-1bc0cfcf7cb1.png)

**Comment by milutinke** [comment]
You can't match Regex with pure Cyrilic characters, you need to use the unicode escape sequence for them.
You can find a list of unicode escape sequence codes for Cyrilic here: https://jrgraphix.net/r/Unicode/0400-04FF
The format of an unicode code is: `\uNUMBERHERE`, example for `ф` is `\u0444`, another example for `д` is `\u0434`
Full word example for `блини` would be `\u0431\u043b\u0438\u043d\u0438`

**Reply by yarosh69** [reply]
![ScreenShot_20221211014832](https://user-images.githubusercontent.com/120141684/206878267-03ed3111-eb47-4003-a74d-72c0e50a627b.png)
![ScreenShot_20221211014904](https://user-images.githubusercontent.com/120141684/206878268-28dfe866-60fd-4f72-a83f-cccf5025d8e7.png)
Doesn't work for me

**Comment by breadbyte** [comment, marked as answer]
You are trying to match a message from MCC.
AutoRespond will not respond to MCC messages.

What you need is a script that triggers on login.

On your MinecraftClient.ini file, you must enable the ScriptScheduler.
![image](https://user-images.githubusercontent.com/14045257/206895622-6d3ee263-1d2b-4c6a-a6ce-7591c7a1cda5.png)

Copy this configuration to the script scheduler.
```
[[ChatBot.ScriptScheduler.TaskList]]
Task_Name = "Autojoin"
Trigger_On_First_Login = false
Trigger_On_Login = true
Trigger_On_Times = { Enable = false, Times = [ ] }
Trigger_On_Interval = { Enable = false, MinTime = 1.0, MaxTime = 10.0 }
Action = "send /jointoserv vanilla"
```

**Reply by milutinke** [reply]
Oh, I've missed that it was from the MCC lol

**Reply by yarosh69** [reply]
I love you thank you very much!!!!!!!!!!!

**Comment by Sasubaysa** [comment]
@yarosh69  how can I reach you ? , İ need help
